Assets/GothicVR/Scripts/Creator/SoundCreator.cs

```diff
@@ -55,6 +55,24 @@ public GameObject Create(PxVobSoundData vobSound, GameObject parent = null)
 
             return soundObject;
         }
+        public GameObject Create(PxVobZoneMusicData vobSound, GameObject parent = null)
+        {
+            var soundObject = new GameObject(vobSound.vobName);
+            soundObject.SetParent(parent);
+
+            var soundObjectCollider = soundObject.AddComponent<BoxCollider>();
+
+            var min = vobSound.boundingBox.min.ToUnityVector();
+            var max = vobSound.boundingBox.max.ToUnityVector();
+
+            soundObject.transform.position = (min + max) / 2f;
```

Comment:
Why is it divided by 2?

Reply:
to find the center of the bounding box as that is needed to properly place the gameobject